docs_src/microservices/configuration/Ch-Configuration.md

```diff
@@ -38,39 +44,30 @@ Taking the `core-data` service as an example:
 -   `./core-data --profile=docker` starts the service using the docker
     profile found locally
 
+* Again, utilizing environment variables for configuration overrides is
+the recommended path as of the Geneva release. Config profiles may be
+deprecated and removed at some point in the future. *
+
 #### Seeding Configuration
 
 When utilizing the registry to provide centralized configuration
 management for the EdgeX Foundry microservices, it is necessary to seed
-the required configuration before starting the services. This is the
-responsibility of the `config-seed` service. The `config-seed` service
-will assume that a service registry is being used and that the necessary
-endpoint is included in the local configuration file. The use of
-profiles is supported by the `config-seed` service in the same manner
-described above.
-
-So for example, if we wanted to seed the registry with docker-related
-configuration information, we could execute the following after starting
-the registry:
-
-`./config-seed --profile=docker`
-
-Assuming a successful run, the `config-seed` will populate the necessary
-values and then exit. In order for a service to now load the
+the required configuration before starting the services. Each service
+has the in-built capability to perform this seeding operation. A service
+will use its local configuration file to initialize the structure and
+relevant values, and then overlay any environment variable override values
+as specified. The end result will be seeded into the service registry if
+such is being used.
+
+In order for a service to now load the
```

Comment:
The flag is replaced with --configProvider or -cp

Reply:
Fixed